CONTRIBUTING.md

```diff
@@ -80,20 +88,62 @@ NB: If you want to change your Meilisearch credentials or the plugins options yo
 Each PR should pass the tests and the linter to be accepted.
 
 ```bash
+curl -L https://install.meilisearch.com | sh # download Meilisearch
+./meilisearch --master-key=masterKey --no-analytics # run Meilisearch
+
 # Tests
-npm run test
+yarn test
 
 # Tests in watch mode
-npm run test:watch
+yarn test:watch
 
 # Linter
-npm run lint
+yarn lint
 
 # Linter with fixing
-npm run lint:fix
+yarn lint:fix
 
 # Build the project
-npm run build
+yarn build
+```
+
+### Run the backfilled-data script
+
+Run the import script using [`npx` (the Node Package Runner)](https://www.npmjs.com/package/npx) via `yarn` (the Node Package Manager).
+- Set up credentials. The import script uses the application's default credentials to communicate with Firebase. Please follow the instructions [generate a private key file for your service account](https://firebase.google.com/docs/admin/setup#initialize-sdk).
+
+- Run the import script interactively via `npx` by running ONE of the following commands:
+  - Run interactively:
+    ```bash
+    npx firestore-meilisearch
+    ```
+
+  - Or run non-interactively with paramaters:
+    ```bash
+    npx firestore-meilisearch \
+      --project <project_id> \
+      --source-collection-path <collection_name> \
+      --index <index_uid> \
+      --batch-size <100/default=300> \
+      --host <host_address> \
+      --api-key <api_key> \
+      --non-interactive
+    ```
+    **Note**: The `--batch-size` and `--query-collection-group` arguments are optional. To see its usage, run the above command with `--help`.
+
+- Run the project for development:
+Launch Meilisearch instance:
+``` bash
+curl -L https://install.meilisearch.com | sh # download Meilisearch
+./meilisearch --master-key=masterKey --no-analytics # run Meilisearch
+```
+Launch the watcher on the project:
+``` bash
+yarn watch
+```
+Launch the watcher on the script. You must first have modified this command in the `package.json` file with your own data:
```

Comment:
```suggestion
Launch the watcher on the script. If you are using different credentials, the informations of the playground script must be changed inside the `package.json` :
```

What do you mean? Which data?

Reply:
I mean your own parameters like your project_id and your collection_path